.devcontainer/devcontainer.json

```
@@ -1,7 +1,7 @@
 {
   "name": "Sample Development Environment for CCF",
   "context": "..",
-  "image": "ccfmsrc.azurecr.io/ccf/ci/sgx:oe-0.18.0",
+  "image": "ccfmsrc.azurecr.io/ccf/ci/sgx:oe-0.18.1-0",
```

```suggestion
  "image": "ccfmsrc.azurecr.io/ccf/ci/sgx:oe-0.18.2-0",
```

Today is not my day.